seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -691,7 +691,7 @@
       ],
       "tests": [
         "assert(quotient === 2.2, 'message: The variable <code>quotient</code> should equal <code>2.2</code>.');",
-        "assert(/\\//.test(code), 'message: You should use the <code>/</code> operator');"
+        "assert(/4\\.4\\s*\\/\\s*2\\.0/.test(code), 'message: You should use the <code>/</code> operator');"
```

Comment:
Hypothetically, I could write `4.400 / 2.0` and it will fail this new test. Other ways we could prevent issues like this is maybe having a regex count the number of `quotient` variables to make sure there is only one? And also maybe have a comment in the code like this

```js
var quotient = 0.0 / 2.0; // Fix this line
```